### ISSUE TITLE
Write pipeline to re-encrypt all input data

### ISSUE THREAD

**Issue by jvivian** [BD2KGenomics/toil-scripts#193]
Old convention uses an https:// url which is zone-specific.   New encryption method concatenates an s3 url to the master key to generate per-file encryption keys. 

ex:   s3://bucket/dir/key

**Comment by jvivian** [comment]
Based on #190 — waiting on merge to submit PR.

**Comment by hannes-ucsc** [comment]
> waiting 

I wouldn't. You can base a new PR on the branch for #190. Once the latter is merged into master, rebasing the new PR on master is very likely to be straight-forward.

**Comment by jvivian** [comment]
> you can base a new PR on the branch for #190

That's what I've done to work on it locally, I just figured it didn't make sense to actually open the PR since it won't be merged until #190 is.

**Comment by jvivian** [comment]
@hannes-ucsc — Should I close this given our discussion with @briandoconnor?  I'll keep the code in my personal repo in case we ever need it.

**Comment by hannes-ucsc** [comment]
Yes
